### ISSUE TITLE
Unexpected error requirement failed: Source file x is a directory

### ISSUE THREAD

**Issue by p41n1** [scalacenter/course-management-tools#297]
Hi,

I noticed that on my Linux machine installing of a course didn't work, since I was able to install the course on my windows I opened an issue here.

What I tried:
```shell
> cmtc install -f -s scalacenter/scala3-migration-course
Downloading studentified course from 'https://github.com/scalacenter/scala3-migration-course/releases/download/v0.1.1/scala3-migration-course-student.zip' to courses directory

ERROR - Failed to execute command.
  Unexpected error(requirement failed: Source file '/tmp/sbt_77588c2b/scala3-migration-course' is a directory.)
  /home/x/.cache/cmt/Courses/scala3-migration-course.zip may be corrupt
```

I investigated the source code a bit and found the following line to be the cause of the problem: https://github.com/lunatech-labs/course-management-tools/blob/main/cmtc/src/main/scala/com/lunatech/cmt/client/command/Install.scala#L84

If I replace the `sbtio.move` line with:
```scala
sbtio.copyDirectory(tmpDir / project, targetFolder)
sbtio.delete(tmpDir / project)
```
the installation works like a charm.

My question: Is this intended or a bug ?

---
Used version:
```shell
> cmtc version
name: Course Management Tools (Client), version: 2.0.19, scalaVersion: 3.3.0, sbtVersion: 1.9.3

> cs java -version
openjdk version "17.0.8.1" 2023-08-24
OpenJDK Runtime Environment (build 17.0.8.1+0-suse-1.1-x8664)
OpenJDK 64-Bit Server VM (build 17.0.8.1+0-suse-1.1-x8664, mixed mode, sharing)

```

**Comment by eloots** [comment]
Thanks for hunting this down!
I'll apply the suggested change.

**Comment by p41n1** [comment]
Hi,

thanks for applying the changes, I noticed the `delete`-call was missing and created a PR.

(Sorry it took so long.. I forgot about the issue)

**Comment by eloots** [comment]
Resolved via #306 and https://github.com/scalacenter/course-management-tools/pull/301

**Comment by eloots** [comment]
@p41n1 Release 2.0.22 contains the fix (including the one for the missing delete)
